### PR TITLE
Fixed some broken links and redirects in docs.

### PR DESCRIPTION
**Pull request by jacobtylerwalls**

### CODE REVIEW

File: docs/topics/i18n/translation.txt [hunk 1]
```diff
@@ -1279,7 +1279,7 @@ You can even pre-generate the JavaScript catalog as part of your deployment
 procedure and serve it as a static file. This radical technique is implemented
 in django-statici18n_.
 
-.. _django-statici18n: https://django-statici18n.readthedocs.io/en/latest/
+.. _django-statici18n: https://django-statici18n.readthedocs.io/
```
Comment (kezabelle) [inline docs/topics/i18n/translation.txt:1282]: Verified, old URL was a 404. New URL currently redirects to `/en/v2.1.0/`

File: docs/topics/install.txt [hunk 2]
```diff
@@ -150,7 +150,7 @@ This is the recommended way to install Django.
         $ python -m pip install Django
 
 .. _pip: https://pip.pypa.io/
-.. _standalone pip installer: https://pip.pypa.io/en/latest/installing/#installing-with-get-pip-py
+.. _standalone pip installer: https://pip.pypa.io/en/latest/installation/
```
Comment (kezabelle) [inline docs/topics/install.txt:153]: Verified, old URL was 'this page has moved' with a soft redirect (`<meta>` or JS) to the new URL given.

File: docs/topics/email.txt [hunk 3]
```diff
@@ -209,7 +209,7 @@ from the request's POST data, sends that to admin@example.com and redirects to
             # to get proper validation errors.
             return HttpResponse('Make sure all fields are entered and valid.')
 
-.. _Header injection: http://nyphp.org/phundamentals/8_Preventing-Email-Header-Injection
+.. _Header injection: http://nyphp.org/phundamentals/8_Preventing-Email-Header-Injection.html
```
Comment (kezabelle) [inline docs/topics/email.txt:212]: Verified, old URL was `Error; We all make them ...`.

File: docs/internals/contributing/committing-code.txt [hunk 4]
```diff
@@ -138,7 +138,7 @@ Django's Git repository:
   Credit the contributors in the commit message: "Thanks A for the report and B
   for review." Use git's `Co-Authored-By`_ as appropriate.
 
-  .. _Co-Authored-By: https://docs.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors
+  .. _Co-Authored-By: https://docs.github.com/en/github/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors
```
Comment (kezabelle) [inline docs/internals/contributing/committing-code.txt:141]: Verified, old URL is a `301` to the new one.

File: docs/faq/help.txt [hunk 5]
```diff
@@ -26,7 +26,7 @@ Then, please post it in one of the following channels:
   chat-based discussions. If you're new to IRC, see the `Libera.Chat
   documentation`_ for different ways to connect.
 
-.. _`"Using Django"`: https://forum.djangoproject.com/c/users
+.. _`"Using Django"`: https://forum.djangoproject.com/c/users/6
```
Comment (kezabelle) [inline docs/faq/help.txt:29]: Verified, old URL is a `301` to the new one.

**Review by kezabelle — APPROVED** [review]

**Comment by felixxm** [comment]
@jacobtylerwalls Thanks :+1: 
@kezabelle Thanks for checking :+1:

**Pull request merged**